charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -303,17 +306,6 @@ data:
                   cpu: {{ .Values.application.ray.cpu }}
                   memory: {{ .Values.application.ray.memory }}Gi
               securityContext:
-              volumeMounts:
```

Comment:
we do not need counts for worker anymore?

Reply:
There were 2 volumeMounts. The latter was used.  This PR merges them into one.